app/views/layouts/mailer.html.erb

```diff
@@ -3,11 +3,175 @@
+      }
+      .mail-wrapper .gravatar {
+        height: 80px;
+        width: 80px;
+        border-radius: 50%;
+        border: 1px solid #ddd;
+      }
+      .mail-wrapper .gravatar img {
+        height: 80px;
+        width: 80px;
+        object-fit: cover;
+        border-radius: 50%;
+      }
+      .mail-wrapper .header {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+      }
+      .mail-wrapper .header.center {
+        justify-content: center;
+      }
+      .mail-wrapper .logo {
+        display: flex;
+        justify-content: center;
+      }
+      .mail-wrapper .logo img {
+        object-fit: cover;
+        height: 80px;
+      }
+      .main {
+        text-align: center;
+      }
+      .main .button {
+        display: inline-block;
+        background-color: #378137; /* #298DCC in pro */
+        border-radius: 5px;
+        padding: 8px 15px;
+        color: #fff;
+        margin: 1rem;
+      }
+      .main .button:hover {
+        background-color: #204b20; /* #2070a2 in pro */
+      }
+      .main .digest-header {
+        font-weight: 300;
+        margin: 1rem 0 1.5rem 0;
+      }
+      .main hr {
+        width: 100%;
+        height: 0;
+        border-top: 1px solid #ddd;
+      }
+      .main .notification {
+        display: flex;
+        align-items: center;
+        margin: 1rem 0;
+      }
+      .main .notification .avatar {
+        height: 50px;
+        width: 50px;
+        border-radius: 50%;
+        border: 1px solid #ddd;
+        margin: 0 20px 0 10px;
+      }
+      .main .notification .avatar img {
+        height: 50px;
+        width: 50px;
+        object-fit: cover;
+        border-radius: 50%;
+      }
+      .main .notification .notification-text {
+        text-align: left;
+        margin: 0;
+        font-weight: 400;
+      }
+      .main .notification .notification-text .user-name {
+        font-weight: 700;
+      }
+      .main .notification .notification-text .time-stamp {
+        font-size: 80%;
+        color: #777;
+        white-space: nowrap;
+      }
+      @media (max-width: 320px) {
+        .main .notification .notification-text .time-stamp {
+          display: block;
+        }
+      }
     </style>
   </head>
 
   <body>
-    <%= yield %>
+    <div class="mail-wrapper">
+
+      <div class="header center">
+        <div class="logo">
+          <%= image_tag 'DradisCE_full_small.png' %> 
+        </div>
+      </div>
+
+      <!-- Dradis Pro Version
+      <div class="header">
+        <div class="logo">
+          <%#= image_tag 'DradisPro_full_small.png' %>
+        </div>
+        <div class="gravatar">
+          <%= image_tag 'profile.jpg' %>
+        </div>
+      </div> -->
+
+      <div class="main">
+        <%= yield %>
+      </div>
+    </div>
+
+    <div class="mail-footer">
+      <div class="row">
+        <a href="javascript:void(0)">Manage your Dradis email preferences</a>
```

Comment:
This being in the default mailer will effect the emails send out by gateway I think. Who don't have preferences or the option to change them. We may need to see what's happening in pro and adjust the gateway layout as well.